shell/platform/tizen/tizen_embedder_engine.cc

```diff
@@ -16,8 +16,37 @@
 // Unique number associated with platform tasks.
 static constexpr size_t kPlatformTaskRunnerIdentifier = 1;
 
+static std::string GetDeviceProfile() {
+  char* feature_profile;
+  system_info_get_platform_string("http://tizen.org/feature/profile",
+                                  &feature_profile);
+  std::string profile(feature_profile);
+  free(feature_profile);
+  return profile;
+}
+
+static std::string GetDeviceModelName() {
```

Comment:
What's the model name for? It looks like `device_model_name` is unused.

Reply:
sometimes, I needed some way to found host device is emulator or not, you can get "Emulator" through this.
But it's definitely not necessary for this PR. Should I remove  it?

Reply:
I think it's better to remove.

Reply:
OK, no problem